src/front/glsl/ast.rs

```diff
@@ -1,12 +1,12 @@
 use super::{super::Typifier, constants::ConstantSolver, error::ErrorKind, SourceMetadata};
 use crate::{
-    proc::ResolveContext, Arena, BinaryOperator, Binding, Block, Constant, Expression, FastHashMap,
-    Function, FunctionArgument, GlobalVariable, Handle, Interpolation, LocalVariable, Module,
-    RelationalFunction, ResourceBinding, Sampling, ShaderStage, Statement, StorageClass, Type,
-    TypeInner, UnaryOperator,
+    front::Emitter, proc::ResolveContext, Arena, BinaryOperator, Binding, Block, Constant,
```

Comment:
there is a `super::super` include in the previous line, let's use it here (i.e. that line is for all the frontend stuff)